CoffeeMachineSimulator/CoffeeMachineSimulator.Services/Services/EspressoMachineService.cs

```diff
@@ -17,13 +17,19 @@ public EspressoMachineService(ICoffeeService coffeeService)
 
         public float GetSumOfAllCoffees()
         {
-            throw new System.NotImplementedException();
+            float sumOfAllPrices= 0f;
+            var coffees = coffeeService.GetCoffees();
+
+            foreach(CoffeeModel i in coffees)
+            { 
+                sumOfAllPrices += i.Price; 
+            }
```

Comment:
there is nothing wrong with using this kind of syntax but you can also use LINQ methods on your list. The only difference would be performance wise (LINQ is a bit slower than a foreach loop, but it is irrelevant in this situation - https://stackoverflow.com/questions/3156059/is-a-linq-statement-faster-than-a-foreach-loop).
You can rewrite the same lines of code with this:

float sumOfAllPrices = coffees.Sum(x=>x.Price);

https://www.csharp-examples.net/linq-sum/